workflow/scripts/create-paths-for-manually-checking.py

```diff
@@ -0,0 +1,42 @@
+sys.stderr = open(snakemake.log[0], "w")
+
+from os import sep
+import pandas as pd
+
+
+def save_df(df: pd.DataFrame, out_path: str):
+    df.to_csv(out_path, sep="\t", index=False, header=False)
+
+
+def no_personal_data_found(summary_df: pd.DataFrame, out_path: str):
+    save_df(summary_df[summary_df["# personal data"] == 0][["processed img"]], out_path)
+
+
+def lots_personal_data_found(summary_df: pd.DataFrame, out_path: str):
+    save_df(
+        summary_df[summary_df["# personal data"] >= 10][["processed img"]], out_path
+    )
+
+
+def address_not_entirely_found(summary_df: pd.DataFrame, out_path: str):
+    save_df(
+        summary_df[summary_df["city"] != summary_df["city"]][["processed img"]],
+        out_path,
+    )
+
+
+def name_not_entirely_found(summary_df: pd.DataFrame, out_path: str):
+    save_df(
+        summary_df[summary_df["name_family"] != summary_df["name_first_0"]][
+            ["processed img"]
+        ],
+        out_path,
+    )
```

Comment:
Not sure if this will work (particularly with the names)

Reply:
So far the results of this query seem okay. Of course, there are some false positives (e.g. the last name occurs in a text), but also false negatives are detected.